DeeployTest/testUtils/typeMapping.py

```
@@ -60,34 +59,36 @@ def parseDataType(name: str) -> type:
 
-def isUnsigned(_input: np.array) -> bool:
-    if (_input).min() < 0:
-        return False
-    return True
+def signPropTypeAndOffset(_type: Type[IntegerImmediate]) -> Tuple[Type[IntegerImmediate], int]:
+    if _type.signed:
+        return _type, 0
 
+    unsigned2signed = {
+        unsigned.typeName: signed for unsigned, signed in zip([t for t in IntegerDataTypes if t.typeMin == 0
+                                                              ], [t for t in IntegerDataTypes if t.typeMin < 0])
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)